parts/linux/cloud-init/artifacts/components.json

```diff
@@ -65,7 +64,24 @@
         },
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=containernetworking/azure-cni",
-          "latestVersion": "v1.6.10",
+          "latestVersion": "v1.5.38",
+          "previousLatestVersion": "v1.5.35",
+          "containerImagePrefetch": {
+            "latestVersion": {
+              "binaries": [
+                "/dropgz"
+              ]
+            },
+            "previousLatestVersion": {
+              "binaries": [
+                "/dropgz"
+              ]
+            }
+          }
+        },
+        {
+          "renovateTag": "registry=https://mcr.microsoft.com, name=containernetworking/azure-cni",
+          "latestVersion": "v1.6.11",
           "previousLatestVersion": "v1.6.7",
```

Comment:
previous latest should be `v1.6.10` here right?

Reply:
v1.6.10 version was not rolledout to all regions in aks-rp. It was rolled out only to specfic subscriptions in few regions.